src/utils/trimWhiteSpaces.test.ts

```diff
@@ -0,0 +1,32 @@
+import trimWhiteSpaces from './trimWhiteSpaces';
+
+describe('trimWhiteSpaces()', () => {
+    let text: string;
+
+    beforeEach(() => {
+        text = 'this is my string';
+    });
+
+    test('should not trip anything extra', () => {
+        const check = trimWhiteSpaces(text);
+        expect(check).toBe('this is my string');
+    });
+
+    test('should trip white space in between the string', () => {
+        text = 'this is my  string';
+        const check = trimWhiteSpaces(text);
+        expect(check).toBe('this is my string');
```

Comment:
I decided to overwrite it instead, thus why it was declared as `let` instead of `const`. I see your point, but I think I'm gonna leave it like this for now because I followed this technique in more places. Maybe in the future I'll change all of them at once.